modules/system/lang/ar/lang.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+return [
+    'app' => [
+        'name' => 'نظام إدارة المحتوى أكتوبر',
+        'tagline' => 'عودة إلى الأساس'
+    ],
+    'locale' => [
+        'ar' => 'العربية',
```

Comment:
You also need to add this to the system default language file (English) so that when a user is on English they can see the proper name for your language.

Reply:
yes, but I'm sorry, how can I add this file to the request?

Reply:
Just add that line to modules/system/lang/en/lang.php

Reply:
Ok, I did in another request.

Reply:
Not in another request, just make the changes in the same branch as this PR: Got to https://github.com/mabehiry/october/blob/mabehiry-patch-1/modules/system/lang/en/lang.php and edit that file and commit it to your branch; then it will show up here.

Reply:
I'm sorry, I did, but it didn't appear here!